.github/workflows/test.yml

```diff
@@ -776,6 +776,92 @@ jobs:
         with:
           commit_message: Benchmark result for ${{ steps.date.outputs.pretty }} (${{ github.sha }})
 
+  next_js_integration:
+    needs: [determine_jobs, rust_prepare, rust_build_release]
+    if: needs.determine_jobs.outputs.rust == 'true' && (github.event_name == 'pull_request' || needs.determine_jobs.outputs.push == 'true')
+    name: Next.js integration test
+    runs-on: ubuntu-latest-8-core-oss
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [16, 18]
+        group: [1, 2]
+    env:
+      # Enabling backtract will makes snapshot tests fail
+      RUST_BACKTRACE: 0
+      NEXT_TELEMETRY_DISABLED: 1
+      NEXT_TEST_JOB: 1
+      __INTERNAL_CUSTOM_TURBOPACK_BINARY: ${{ github.workspace }}/artifacts/release-x86_64-unknown-linux-musl/next-dev
+
+    steps:
+      # https://github.com/actions/virtual-environments/issues/1187
+      - name: tune linux network
+        run: sudo ethtool -K eth0 tx off rx off
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: vercel/next.js
+          ref: v13.0.5-canary.5
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 7.3.0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: pnpm
+
+      - name: Rust Cache
```

Comment:
We don't compile rust so we don't need that?

Reply:
yes, that's correct.